spring-thrift-starter/src/main/java/ru/trylogic/spring/boot/thrift/beans/RequestIdLogger.java

```diff
@@ -38,7 +39,7 @@ protected String generateXRequestId() {
 
   protected String getXRequestId(HttpServletRequest request) {
     String requestId = request.getHeader(getRequestIdHeader());
-    if (Strings.isNullOrEmpty(requestId)) {
+    if (StringUtils.isEmpty(requestId)) {
```

Comment:
You can use StringUtils from Spring (you won't have to take commons as a dependency then)